### DISCUSSION TITLE
How to downlaod and install fooocus successfully on Window

### DISCUSSION THREAD

**Discussion by aborisade2424** [lllyasviel/Fooocus#3616] [Q&A] [answered]
Please can any one explain to me how to install fooocus on my window laptop
i clicked the download link form the website and it does come with the installer 
i couldnt not unzip the file that was downloaded as it shows like a notepad

**Comment by ILikeToasters** [comment, marked as answer]
To extract the .7z file you downloaded you need to install the 7zip program. You can check out this video. It is for an earlier version but the install process has not changed. https://youtu.be/j1WuQndmgFE
